pkg/assume/assume.go

```diff
@@ -60,6 +61,12 @@ func AssumeCommand(c *cli.Context) error {
 		var p string
 		err = testable.AskOne(&in, &p, withStdio)
 		if err != nil {
+			if strings.Contains(err.Error(), "please provide options") {
```

Comment:
I think this check should go before the prompt.

We can check if the list of profiles is empty instead